apps/src/componentLibrary/alert/alert.module.scss

```diff
@@ -19,50 +19,50 @@ $alert-span-vertical-top-spacing: 2px;
     }
 
     a {
-      color: $light_black;
+      color: var(--neutral-base-black);
     }
   }
 }
 
 // Types (colors)
 .alert-primary {
-  background-color: $light_secondary_100;
+  background-color: var(--brand-purple-10);
 }
 
 .alert-success {
-  background-color: $light_affirmative_100;
+  background-color: var(--sentiment-success-10);
 
   .alertContentContainer i {
-    color: $light_affirmative_500;
+    color: var(--sentiment-success-50);
```

Comment:
Example of correct variable name showing in DevTools:
<img width="1373" alt="Screenshot 2024-11-27 at 12 06 32 PM" src="https://github.com/user-attachments/assets/14e18d47-f510-42ae-bf1c-98dcae79f402">